cmd/limactl/copy.go

```diff
@@ -128,14 +154,28 @@ func copyAction(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
+
 	sshArgs := sshutil.SSHArgsFromOpts(sshOpts)
 
-	sshCmd := exec.Command(arg0, append(sshArgs, scpArgs...)...)
+	sshCmd := exec.Command(arg0, createArgs(sshArgs, copyToolArgs, defaultTool)...)
 	sshCmd.Stdin = cmd.InOrStdin()
 	sshCmd.Stdout = cmd.OutOrStdout()
 	sshCmd.Stderr = cmd.ErrOrStderr()
-	logrus.Debugf("executing scp (may take a long time): %+v", sshCmd.Args)
+	logrus.Debugf("executing %s (may take a long time): %+v", arg0, sshCmd.Args)
```

Comment:
Here we can log the cmd instance instead of arg0 and rest fo the args.

It will be easier to make this change if we work first on preparing the code to add a rsync option. Improve the logs, variables names, and orgnization so adding a second option is easy in the next step.